pkg/users/account-create-dialog.js

```diff
@@ -88,6 +105,31 @@
                     </FlexItem>
                 </Flex>
             </FormGroup>
+            <FormGroup label={_("Home directory")}
+                       helperTextInvalid={errors && errors.home_dir}
+                       validated={(errors && errors.home_dir) ? "error" : "default"}
+                       fieldId="accounts-create-user-home-dir">
+                <TextInput id="accounts-create-user-home-dir"
+                    onChange={value => change("home_dir", value)}
+                    placeholder={_("Path to directory")}
+                    value={state.home_dir} />
+            </FormGroup>
+            <FormGroup label={_("Shell")}
+                       fieldId="accounts-create-user-shell">
+                <Select variant={SelectVariant.single}
+                        toggleId="accounts-create-user-shell"
+                        onToggle={statusIsExpanded => change("isShellSelectExpanded", statusIsExpanded)}
+                        onSelect={(event, selection) => { change("shell", selection); change("isShellSelectExpanded", false) }}
+                        selections={shell}
+                        isOpen={isShellSelectExpanded}
+                        aria-labelledby="vm-state-select"
+                        menuAppendTo="parent"
+                        isCreatable
+                        isCreateOptionOnTop
+                        createText={_("Use")}>
+                    { shells.map(shell_path => <SelectOption value={shell_path} key={shell_path} label={shell_path}>{shell_path}</SelectOption>) }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230202-125519-871b841d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)